finetune.py

```diff
@@ -0,0 +1,184 @@
+"""Finetuning loop."""
+
+import os
+import logging
+import argparse
+import time
+
+import torch
+from orb_models.forcefield import pretrained
+from orb_models.finetune_utilities import experiment, optim
+from orb_models.dataset import data_loaders
+from orb_models.finetune_utilities import steps
+
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+
+def run(args):
+    """Training Loop.
+
+    Args:
+        config (DictConfig): Config for training loop.
+    """
+    device = experiment.init_device()
+    experiment.seed_everything(args.random_seed)
+
+    # Make sure to use this flag for matmuls on A100 and H100 GPUs.
+    torch.set_float32_matmul_precision("high")
+
+    # Instantiate model
+    model = pretrained.orb_v1(device=device)
+    for param in model.parameters():
+        param.requires_grad = True
+    model_params = sum(p.numel() for p in model.parameters() if p.requires_grad)
+    logging.info(f"Model has {model_params} trainable parameters.")
+
+    # Move model to correct device.
+    model.to(device=device)
+    total_steps = args.max_epochs * args.num_steps
+    optimizer, lr_scheduler = optim.get_optim(args.lr, total_steps, model)
```

Comment:
Move this get_optim function either inline, or in a function in this file, once the param_groups code has been removed

Reply:
moved `get_optim` to experiment.py